### PR TITLE
#16393 Add lasso selection tool

### LINKED ISSUE #16393: selecting geometries by drawing an area on a map
Is it possible to select geometries e.g. by drawing a rectangle on a map (like on the middle of the map below) and select everything inside the selected area?
afbeelding
The boundary of the drawn area can be used in a gis function that is added to the filter clause
(e.g. _**st_contains((ST_MakeEnvelope(123456, 456789, 133456, 466789, 28992)), the_geom**_) ).
I do not know if the gis-functions that are needed for this are universal (I only use postgres). An alternative could be getting the id's of the selected records (if a table has a primary key) and add these id's to the filter.

### PR DESCRIPTION
**Pull request by ShadelessFox**
This PR also removes duplicate version of the library we already have a minimized version of.

https://user-images.githubusercontent.com/35821147/202279325-52bed6ac-5154-4e2f-9f8b-e587d3344161.mp4

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.data.gis.view/web/inc/leaflet-lasso.min.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
...(e.geometry):null;case"FeatureCollection":return p(e);case"GeometryCollection":return f(e);default:throw new Error("Unknown type: "+e.type)}return null},h=function(t){return!isNaN(parseFloat(t))&&isFinite(t)},g=function(t,e,o,n){var r=(n[0]-o[0])*(t[1]-o[1])-(n[1]-o[1])*(t[0]-o[0]),i=(e[0]-t[0])*(t[1]-o[1])-(e[1]-t[1])*(t[0]-o[0]),s=(n[1]-o[1])*(e[0]-t[0])-(n[0]-o[0])*(e[1]-t[1]);if(0!==s){var a=r/s,l=i/s;if(a>=0&&a<=1&&l>=0&&l<=1)return!0}return!1},y=function t(e,o){if(h(e[0][0])){if(h(o[0][0])){for(var n=0;n<e.length-1;n++)for(var r=0;r<o.length-1;r++)if(g(e[n],e[n+1],o[r],o[r+1]))return!0}else for(var i=0;i<o.length;i++)if(t(e,o[i]))return!0}else for(var s=0;s<e.length;s++)if(t(e[s],o))return!0;return!1},v=function(t,e){for(var o=!1,n=-1,r=t.length,i=r-1;++n<r;i=n)(t[n][1]<=e[1]&&e[1]<t[i][1]||t[i][1]<=e[1]&&e[1]<t[n][1])&&e[0]<(t[i][0]-t[n][0])*(e[1]-t[n][1])/(t[i][1]-t[n][1])+t[n][0]&&(o=!o);return o},m=function(t,e){for(var o=0;o<t.length;o++)if(t[o]!==e[o])return!1;return!0},M=function(t,e){return t[0]>e[0]?-1:t[0]<e[0]?1:t[1]>e[1]?-1:t[1]<e[1]?1:0},L=function(t){for(var e=[],o=0;o<t.length;o++){var n=t[o].slice();!1===m(n[0],n[n.length-1])&&n.push(n[0]),e.push(n)}return e};
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.data.gis.view/web/inc/leaflet-lasso.min.js:1]: Can it be formatted?

Comment (ShadelessFox) [reply plugins/org.jkiss.dbeaver.data.gis.view/web/inc/leaflet-lasso.min.js:1]: No, it's minimized for a reason.

Comment (LonwoLonwo) [reply plugins/org.jkiss.dbeaver.data.gis.view/web/inc/leaflet-lasso.min.js:1]: sad

Comment (ShadelessFox) [reply plugins/org.jkiss.dbeaver.data.gis.view/web/inc/leaflet-lasso.min.js:1]: It's just a dependency. Neither you nor I am supposed to really care about how pretty it is. Minimization reduces size, and this is what actually matters.

**Review by E1izabeth — APPROVED** [review]

File: plugins/org.jkiss.dbeaver.data.gis.view/web/inc/leaflet-lasso.min.js (hunk 1 above)
Comment (LonwoLonwo) [reply plugins/org.jkiss.dbeaver.data.gis.view/web/inc/leaflet-lasso.min.js:1]: Ok. And what about "wkx.js"? Why you deleted it?

Comment (ShadelessFox) [reply plugins/org.jkiss.dbeaver.data.gis.view/web/inc/leaflet-lasso.min.js:1]: We were bundling both unminimized and minimized versions of the same library. Please see individual commits,

**Comment by Matvey16** [comment]
Verified

**Pull request merged**